.env.test.example

```
@@ -1,15 +1,21 @@
-# Test environment specific overrides
+# Application
 ENVIRONMENT=test
 DJANGO_DEBUG=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DJANGO_DEBUG key should go before the ENVIRONMENT key

(UnorderedKey)